docs/index.rst

```diff
@@ -6,6 +6,7 @@ Qiskit Experiments documentation
   :maxdepth: 2
 
   API References <apidocs/index>
+  Experiment Library <apidocs/library>
```

Comment:
This means base class API docs will not be shown because subclasses have documentation for each method?

Reply:
They might still be shown in other parts of documentation, but this is just to give a easy to find list of experiments which is what most people will care about